src/include/override/sys/syscall.h

```diff
@@ -91,6 +97,10 @@
 #    endif
 #  elif defined(__s390__)
 #    define systemd_NR_close_range 436
+#  elif defined(__sh__)
+#    define systemd_NR_close_range 436
+#  elif defined(__sh64__)
+#    define systemd_NR_close_range -1
```

Comment:
Hmm, negative one sounds spurious and dangerous. It seems the sh64 table is outdated and not list recent syscalls.

Reply:
https://github.com/hrw/syscalls-table/issues/125

Reply:
I don't know anything about `sh64`. I guess we could assume that those missing syscalls are the same as on `sh` and just manually adjust the table until hrw gets the chance to respond.

Reply:
See https://github.com/torvalds/linux/commit/45799491a92174ff78d9c46de55d614814bdd3e1 and https://github.com/torvalds/linux/commit/37744feebc086908fd89760650f458ab19071750.

Reply:
So, please drop sh64 things.

Reply:
already did